### PR TITLE
I18n pt br

### PR DESCRIPTION
**Pull request by LeonardoFerreiraa**
Translate all texts of the application to pt-BR

and a awesome cat picture

image

### CODE REVIEW

File: app/views/vacancies/show.html.erb [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
               <tr>
                 <td><span class="vacancy-describe-itens"> <i class="fa fa-calendar" aria-hidden="true"></i> </span></td>
-                <td><%= local_time_ago(@vacancy.created_at) %></td>
+                <td><%= I18n.l(@vacancy.created_at, format: :short) %></td>
```
Comment (duduribeiro) [inline app/views/vacancies/show.html.erb]: Keep using local_time_ago. If it is recent, it keeps the message like `x minutes ago`, `yesteday`. This stuffs

File: config/locales/pt-BR.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      welcome_message_html: "Encontre as melhores vagas! <br> Encontre os profissionais certos para sua empresa!"
+    show:
+      apply_help_html: >
+                 <li>  Encontre uma vaga, pesquise seus requisitos e garata que os cumpra.</li>
+                 <li>  Escreva seu currículo e uma carta de apresentação.</li>
+                 <li>  Pegue algumas referências de experiências anteriores.</li>
+                 <li>  Candidate-se online ou por E-Mail.</li>
+                 <li>  Após uma semana, entre em contato com o responsável pela contratação.</li>
+      help_friends:    "Você pode ajudar seus amigos a encontrar ótimas oportunidades!"
+      how_to_apply:    "Como candidatar-se"
```
Comment (duduribeiro) [inline config/locales/pt-BR.yml]: "Como se candidatar", lets keep it informal

File: config/locales/pt-BR.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+pt-BR:
+  menu:
+    all_vacancies: "Todas as vagas"
+  messages:
+    apply_for_job: "Candidatar-se"
```
Comment (duduribeiro) [inline config/locales/pt-BR.yml]: "Se candidatar" or "Aplicar"

File: config/locales/pt-BR.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      advertise_message:    "Anunciar vaga"
+      no_vacancies:         "Desculpe, nenhuma vaga disponível 😔"
+      title:                "Todas as vagas"
+      welcome_message_html: "Encontre as melhores vagas! <br> Encontre os profissionais certos para sua empresa!"
+    show:
+      apply_help_html: >
+                 <li>  Encontre uma vaga, pesquise seus requisitos e garata que os cumpra.</li>
+                 <li>  Escreva seu currículo e uma carta de apresentação.</li>
+                 <li>  Pegue algumas referências de experiências anteriores.</li>
+                 <li>  Candidate-se online ou por E-Mail.</li>
```
Comment (duduribeiro) [inline config/locales/pt-BR.yml]: "Se candidate"

File: spec/features/register_vacancy_spec.rb [hunk 5]
```diff
@@ -12,12 +12,12 @@
     fill_in I18n.t("vacancies.new.how_to_apply"), with: "Send an email to hr@theempire.com with your CV and a cover letter."
 
     within("#company-details") do
-      fill_in "Name",   with: "The Empire"
-      fill_in "URL",    with: "www.theempire.com"
-      fill_in "E-mail", with: "hr@theempire.com"
+      fill_in I18n.t("vacancies.new.company_name"),   with: "The Empire"
```
Comment (duduribeiro) [inline spec/features/register_vacancy_spec.rb]: 💯

File: app/views/vacancies/show.html.erb (hunk 1 above)
Comment (LeonardoFerreiraa) [inline app/views/vacancies/show.html.erb]: `local_time_ago` does not support i18n :/

**Pull request merged**